src/internal/operators/tap.ts

```diff
@@ -81,16 +81,16 @@ export function tap<T>(
  *
  * ```ts
  * import { of, interval } from 'rxjs';
- * import { tap, concatMap, take } from 'rxjs';
+ * import { tap, map, concatMap, take } from 'rxjs/operators';
  *
  *
  * of(1, 2, 3).pipe(
- *  concatMap(n => interval.pipe(
+ *  concatMap(n => interval(1000).pipe(
  *    take(Math.round(Math.random() * 10)),
  *    map(() => 'X'),
- *    tap({
+ *    tap(n => ({
  *      complete: () => console.log(`Done with ${n}`)
- *    })
+ *    }))
```

Comment:
This last bit isn't an error. An observer with only a `complete` method is being passed.